crates/bevy_internal/src/prelude.rs

```diff
@@ -39,10 +43,6 @@ pub use crate::animation::prelude::*;
 #[cfg(feature = "bevy_color")]
 pub use crate::color::prelude::*;
 
-#[doc(hidden)]
-#[cfg(feature = "bevy_core_pipeline")]
-pub use crate::core_pipeline::prelude::*;
-
```

Comment:
Note: this prelude is obsolete, because it only re-exported camera types which are now covered by bevy_camera, but left in the crate itself because removing it would be a breaking change. It will yeet in 0.18